src/chapters/typescript/studio.rst

```diff
@@ -169,14 +220,15 @@ Expected Console Output
    Water true
    Food true
    Tesla Roadster false
-   final cargo and astronaut weight: 5656.78
-
+   final cargo and astronaut mass: 5656.78
 
 Submitting Your Work
---------------------
-
-In Canvas, open the TypeScript studio and click the "Submit" button. An input
-box will appear.
-
-Copy the URL for your repl.it project and paste it into the box, then click
-"Submit" again.
+---------------------
+
+#. Once you have your project working, use the terminal to commit and push your
+   work up to GitHub.
```

Comment:
maybe worth it to say 'your GitHub repo' so as to avoid confusing them with the upstream LaunchCode education origin of the fork?

Reply:
If they cloned the repo from GitHub to local, then pushing should automatically save to their account, but I agree that the phrasing could be more direct.

Fixed.